block-explorer/v3.yml

```diff
@@ -0,0 +1,589 @@
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /block/{hash}:
+    get:
+      tags:
+        - Block
+      summary: Get block by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of block to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Block for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Block'
+
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /transaction/{hash}:
+    get:
+      tags:
+        - Transaction
+      summary: Get transaction by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of transaction to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Transaction for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Transaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/latest/transaction:
+    get:
+      tags:
+        - Transaction
+      summary: Get transactions from the latest snapshot
+      parameters:
+        - name: limit
+          in: query
+          description: Limit (size) of the response items
+          required: false
+          schema:
+            type: integer
+        - name: search_after
+          in: query
+          description: ISOString to search after given timestamp. Please keep in mind that it's mutually exclusive with search_before.
+          required: false
+          schema:
+            type: string
+            title: ISOString
+        - name: search_before
+          in: query
+          description: ISOString to search before given timestamp. Please keep in mind that it's mutually exclusive with search_after.
+          required: false
+          schema:
+            type: string
+            title: ISOString
+      responses:
+        200:
+          description: Transactions from the latest snapshot
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{hash}/transaction:
+    get:
+      tags:
+        - Transaction
+      summary: Get transactions for the given snapshot
```

Comment:
Maybe: `Get transactions for the given snapshot identified by hash!` Not sure it adds much, just thinking out loud.

Reply:
I'll keep that `for the given global snapshot`. There is only one parameter so it shouldn't confuse